src/components/home/BusinessesForm.js

```diff
@@ -11,13 +11,130 @@ class BusinessesForm extends Component {
       searchText: '',
       showDropdown: false,
       showPreviewDropdown: false,
+      actualSelectedItem: null,
+      currentIndex: null,
+      keyPress: false,
     };
+    this.defaultList = null;
+    this.resultsListHTML = null;
+    this._formOnKeyDown = this._formOnKeyDown.bind(this);
+    this._onMouseOver = this._onMouseOver.bind(this);
+    this._fromNodeListToArray = this._fromNodeListToArray.bind(this);
+    this._move = this._move.bind(this);
+    this.noop = this.noop.bind(this);
+    this._onMouseMove = this._onMouseMove.bind(this);
   }
+
+  noop() {}
+
+  _onMouseMove() {
+    this.setState({keyPress: false});
+  }
+
+  _formOnKeyDown(event) {
+    switch (event.key) {
+      case 'Enter':
+        event.preventDefault();
+        if (this.state.actualSelectedItem) {
```

Comment:
`actualSelectedItem` should be deconstructed from `this.state` because it's used twice.